app/views/invitations/new.html.haml

```diff
@@ -0,0 +1,14 @@
+%section.invite.container
+  .row
+    .col-sm-10.col-sm-offset-1
+      = bootstrap_form_for @invitation, class: 'invite' do |f|
+        %header
+          %h1 Invite a friend to join MyFlix!
+        %fieldset
+          = f.text_field :recipient_name, label: "Friend's name"
+        %fieldset
+          = f.email_field :recipient_email, label: "Freind's Email Address"
+        %fieldset
+          = f.text_area :message, rows: 6, label: "Message", placeholder: "I'm using Myflix, you can join today too"
+        %fieldset.form-group.action
+          %input(type="submit" value="Send Invitation" class="btn btn-default")
```

Comment:
I am not sure I understand what you mean, but here would be my guess:

You are using `bootstrap_form_for` - this is from special gem, which produces slightly upgraded html code (enriched with bootstrap classes/tags etc). So You would have to take a look at the bootstrap form documentation and try to check the produced code in browser too. Then work with bootstrap official docs to adjust to whatever you need. It takes a little bit of work but this general approach is what we want you to go through, because you will face these situations often in development.

Another guess: the field set tag above might be redundant.